dockstore-client/src/test/resources/YamlVerifyTestDirectory/correct-directory/tool/.dockstore.yml

```diff
@@ -1,6 +1,6 @@
 version: 1.2
 tools:
-  -  subclass: WDL
+  -  subclass: CWL
```

Comment:
Is this intentional? The `primaryDescriptorPath` is a WDL. If intentional, could use a comment in cwl file (think they use `doc`